js/stores/wizard_store.js

```diff
@@ -436,11 +437,17 @@ const useRawWizardStore = create((
       .then((data) => {
         if (!data.submission_id) {
           log(data);
+          set(withCapturedHistory({
```

Comment:
If the page doesn't change, do we want a new history entry? I'd think this would force the user to use the back button twice to go back.